iele-semantics-plugin/MANTIS.ml

```diff
@@ -0,0 +1,35 @@
+open Constants
+open Constants.K
+open World
+
+module Cache = Caching.Make(World.InMemoryWorldState)
+
+let hook_getBalance c _ _ config _ = match c with
```

Comment:
Could you kindly remind me what are the five arguments to the function hook_getBalance?

Reply:
arguments, klabel, return sort, configuration, fresh function. This is the signature of all hooks in the OCAML backend, but most of these are not used by these hooks.